src/integrationtests/java/util/IntegrationTestBase.java

```diff
@@ -153,13 +154,19 @@ protected void sendEventsAndConfirm() throws Exception {
                     TimeUnit.MILLISECONDS.sleep(firstEventWaitTime);
                     alreadyExecuted = true;
                 }
+                /**
+                 * Without a small delay between the sending of 2 events, one may risk to be lost in
+                 * an empty void of not received by EI
+                 */
+                TimeUnit.MILLISECONDS.sleep(DEFAULT_DELAY_BETWEEN_SENDING_EVENTS);
```

Comment:
Adding sleep is not a guarantee things will work. Travis is usually slower than our local computers. Isn't there any possibility to check that the events have been sent?

Reply:
Thought of this as well, and have now created a new issue since it is not only integration tests that has this isue.
https://github.com/eiffel-community/eiffel-intelligence/issues/403

Reply:
I guess the waitForEventsToBeProcessed method does something like that.